main.py

```diff
@@ -283,13 +283,17 @@ def run_experiment(
             if accumulation_steps is None:
                 accumulation_steps = 1
 
+    # TODO this causes errors because model_language_mapping is not in mate_info
```

Comment:
@Stern5497 are you sure this is the case? This field added, see: https://github.com/JoelNiklaus/LEXTREME/blob/main/utils/utilities.py#L41